itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenarios.java

```diff
@@ -1316,7 +1307,7 @@ public NotificationEventResponse apply(@Nullable NotificationEventResponse event
     eventTypeValidator.assertInjectionsPerformed(true,false);
     InjectableBehaviourObjectStore.resetGetNextNotificationBehaviour(); // reset the behaviour
 
-    verifyRun("SELECT a from " + replDbName + ".ptned where (b=1) ORDER BY a", ptn_data, driverMirror);
+    verifyIfTableNotExist(replDbName , "ptned", metaStoreClientMirror);
```

Comment:
There are other test cases to see if replication works for dynamic inserts. So, this test case is intended to check if INSERT event is not generated for dynamic inserts.